examples/clojure/Earthfile

```diff
@@ -0,0 +1,17 @@
+VERSION 0.7 
+FROM clojure:temurin-8-lein
+WORKDIR /var/app
+
+ARG --global CLOJURE_STANDALONE_ARTIFACT="target/uberjar/earthly-example-0.1.0-SNAPSHOT-standalone.jar"
```

Comment:
It's generally better to avoid using `ARG --global` since changing this value will change the cache key hash of all targets (which will bust the cache).

I wonder if this `ARG` is even needed anywhere? Instead you could use a wildcard, e.g.

```
VERSION 0.7

a:
  FROM alpine
  RUN mkdir -p target/uberjar/
  RUN echo "bogus data" > target/uberjar/earthly-example-0.1.0-SNAPSHOT-standalone.jar
  SAVE ARTIFACT target/uberjar/earthly-example-*-SNAPSHOT-standalone.jar earthly-example.jar

b:
  FROM alpine
  COPY +a/earthly-example.jar .
  RUN cat earthly-example.jar
```

Additionally, is there a way you can extract the version (e.g. `0.1.0`) from the jar? if so, you could then bake in the version into the docker tag, e.g.

```
ARG tag="$(./extract-version earthly-example.jar)"
SAVE IMAGE clojure-example:$tag
```